CHANGELOG.md

```diff
@@ -30,7 +36,6 @@
   * PayPal
     * Make `PayPalPaymentAuthRequestParams` internal
     * Remove `PayPalPaymentAuthResultInfo`
-    * Rename `PayPalLauncher.handleReturnToAppFromBrowser()` to `PayPalLauncher.handleReturnToApp()`
```

Comment:
Should this be removed?

Reply:
Good catch, it was removed at some point. Reverted it, thanks! [0f21f4d](https://github.com/braintree/braintree_android/pull/1174/commits/0f21f4df557fb4556a4b6a756e76508bd3bbb071)